Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.11
+
+WORKDIR /go/src/github.com/zwzn/comicbox/
+
+# Setup the build dependencies
+RUN curl -sL https://deb.nodesource.com/setup_11.x | bash -
+RUN apt-get install -y nodejs
+RUN go get github.com/zwzn/go-bindata/go-bindata
+
+# Download js dependencies
+COPY makefile .
+COPY web/package.json web/package-lock.json web/
+RUN cd web && npm install && cd .. 
+
+# Download go dependencies and compile
+# TODO: Figure out a way to cache the go dependencies like we do the JS ones to speed up build
+COPY . .
+RUN make npm \
```

Comment:
Don't need this `make npm` anymore with the changes to the makefile, I'll remove it